lib/modules/syntaxHighlighting.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+
+import hljs from 'highlight.js';
+import { Module } from '../core/module';
+
+export const module: Module<*> = new Module('syntaxHighlighting');
+
+module.moduleName = 'syntaxHighlightingName';
+module.category = 'appearanceCategory';
+module.description = 'syntaxHighlightingDesc';
+module.options = {
+	auto: {
+		title: 'syntaxHighlightingEnableTitle',
+		type: 'boolean',
+		value: true,
+		description: 'syntaxHighlightingEnableDesc',
+		bodyClass: true,
+	},
+};
+
+module.contentStart = () => {
+	if (module.options.auto.value) {
+		hljs.highlightAll()
```

Comment:
It probably isn't that big of a concern, but if this is done on a page with lots of comments and lots of code blocks it might lock up the page for a bit. 

It could be that hljs already takes care of that, but if that isn't the case it might be worthwhile to take a slightly different approach. Something along the lines of first selecting all visible things on page and highlighting them individually through [`forEachChunked`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/7b7341cf2c897e7c7538180cd8a6e8b70b4c5671/lib/utils/async.js#L26) (there might be a better utility function for this but my RES is a bit rusty).


Another but also minor issue is comments that get dynamically loaded in later (load more comments), I don't think this will highlight those.